manifests/install/repos/yum.pp

```diff
@@ -20,23 +20,30 @@
     default   => $gpgcheck_enabled_default,
   }
   yumrepo { $name:
-    descr    => "Foreman ${repo} repository",
+    descr    => "Foreman ${repo}",
     baseurl  => "http://yum.theforeman.org/${repo_path}/${yumcode}/\$basearch",
     gpgcheck => $gpgcheck_enabled,
     gpgkey   => "http://yum.theforeman.org/${repo_path}/RPM-GPG-KEY-foreman",
     enabled  => '1',
   }
+  yumrepo { "${name}-source":
+    descr    => "Foreman ${repo} - source",
+    baseurl  => "http://yum.theforeman.org/${repo_path}/${yumcode}/source",
+    gpgcheck => $gpgcheck_enabled,
+    gpgkey   => "http://yum.theforeman.org/${repo_path}/RPM-GPG-KEY-foreman",
+    enabled  => '0',
+  }
   yumrepo { "${name}-plugins":
-    descr    => "Foreman ${repo} plugins repository",
+    descr    => "Foreman plugins ${repo}",
     baseurl  => "http://yum.theforeman.org/${plugins_repo_path}/${yumcode}/\$basearch",
     gpgcheck => '0',
     enabled  => '1',
   }
-  yumrepo { "${name}-source":
-    descr    => "Foreman ${repo} source repository",
-    baseurl  => "http://yum.theforeman.org/${repo_path}/${yumcode}/source",
+  yumrepo { "${name}-plugins-source":
+    descr    => "Foreman plugins ${repo} - source",
+    baseurl  => "http://yum.theforeman.org/${plugins_repo_path}/${yumcode}/source",
     gpgcheck => $gpgcheck_enabled,
-    gpgkey   => "http://yum.theforeman.org/${repo_path}/RPM-GPG-KEY-foreman",
+    gpgkey   => "http://yum.theforeman.org/${plugins_repo_path}/RPM-GPG-KEY-foreman",
```

Comment:
I now see that the release files use ```gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman``` in https://github.com/theforeman/foreman-packaging/blob/rpm/develop/foreman/foreman-plugins.repo. Should we just set the gpgkey to the http version but gpgcheck off?